scripts/lightweight_curator.py

```diff
@@ -84,7 +85,7 @@ def get_max_allowed_size(es, percentage_threshold):
     for node in data.splitlines():
         i = i + int(node)
 
-    max_allowed_size = (percentage_threshold * i) / 100.0
+    max_allowed_size = int( (percentage_threshold * i) / 100.0 )
```

Comment:
Why should this be int?
I don't remember the functionality but in some languages converting 78.5 to int returns 78
If this is intentional ingore

Reply:
On bellow line I had previously _limit=int(max_allowed_size)_ for the reason that units are already in bytes. And I am deleting based on bytes. Imho there is no reason for displaying max_allowed_size as float. Just round the max_allowed_size.

`log(log_info, "Do not add into actionable list: '{index}', summed disk usage is {usage} B and disk limit is {limit} B".format(
                index=index.name, usage=indices_size_counter, limit=max_allowed_size))`

I would still like to know your opinion on this Ron.

Reply:
I stand corrected, and thanks for the elaborate response.
This should be in bytes and shouldn't be a float